src/bin/main.rs

```diff
@@ -17,30 +18,13 @@ pub struct Args {
     /// Path to config file.
     #[arg(short, long)]
     pub config: Option<String>,
-    /// Loads trusted root certificates from a file. The file should contain a sequence of PEM-formatted CA certificates.
-    #[arg(long)]
-    pub ca_root_file: Option<String>,
-    /// Trust invalid certificates. This introduces significant vulnerabilities, and should only be used as a last resort.
-    #[arg(long)]
-    pub danger_accept_invalid_certs: bool,
-    /// If a spatial table has SRID 0, then this default SRID will be used as a fallback.
-    #[arg(short, long)]
-    pub default_srid: Option<i32>,
-    #[arg(help = format!("Connection keep alive timeout. [DEFAULT: {}]", ConfigBuilder::KEEP_ALIVE_DEFAULT),
-              short, long)]
-    pub keep_alive: Option<usize>,
-    #[arg(help = format!("The socket address to bind. [DEFAULT: {}]", ConfigBuilder::LISTEN_ADDRESSES_DEFAULT),
-          short, long)]
-    pub listen_addresses: Option<String>,
-    #[arg(help = format!("Maximum connections pool size [DEFAULT: {}]", ConfigBuilder::POOL_SIZE_DEFAULT),
-          short, long)]
-    pub pool_size: Option<u32>,
-    /// Scan for new sources on sources list requests
+    /// [Deprecated] Scan for new sources on sources list requests
```

Comment:
Is the watch flag deprecated? If yes then a warning should be printed.

Reply:
This flag is hidden from help. There is a depreciation notice if it gets used